README.md

```diff
@@ -14,40 +14,123 @@ For an example config look at `example.nix`
 
-`vim krebs/3modules/external/default.nix`
+```
+$EDITOR krebs/3modules/external/default.nix
+```
 
-add your host nix information to this file, please mind the alphabetical sorting
+Prepare a pull-request:
 
-now, create a patch out of your changes:
+```
+git add krebs/3modules/external/default.nix
+git commit -m 'external: add myhostname'
+git format-patch origin/master..
+```
 
-`git diff > retiolum.patch`
+This will print the file name of the formatted patch, which will be called
+something like `0001-external-add-myhostname.patch` (depending on your commmit
+message).  The the contents of that should look something like this:
+
+```patch
+From 75785902b71f03474c446694c5e1e25cd8c3ee23 Mon Sep 1700:00:00 2001
+From: myname <myname@mydomain>
+Date: Wed, 11 Sep 2019 11:34:44 +0200
+Subject: [PATCH] external: add myhostname
+
+---
+ krebs/3modules/external/default.nix | 17 +++++++++++++++++
+ 1 file changed, 17 insertions(+)
+
+diff --git a/krebs/3modules/external/default.nix b/krebs/3modules/external/default.nix
+index aac67f2e..b6cdaebc 100644
+--- a/krebs/3modules/external/default.nix
++++ b/krebs/3modules/external/default.nix
+@@ -393,6 +393,20 @@ in {
+         };
+       };
+     };
++    myhostname = {
++      owner = config.krebs.users.myuser;
++      nets = {
++        retiolum = {
++          ip4.addr = "10.243.my.ip";
++          aliases = [ "myhostname.r" ];
++          tinc.pubkey = ''
++            -----BEGIN RSA PUBLIC KEY-----
++            ...
++            -----END RSA PUBLIC KEY-----
++          '';
++        };
++      };
++    };
+     qubasa = {
+       owner = config.krebs.users.qubasa;
+       nets = {
+@@ -693,6 +707,9 @@ in {
+       mail = "joerg@thalheim.io";
+       pubkey = ssh-for "Mic92";
+     };
++    myuser = {
++      mail = "myuser@mydomain";
++    };
+     qubasa = {
+       mail = "luis.nixos@gmail.com";
+     };
+-- 
+2.19.2
+```
 
-## ask for approval
+## ask for the patch to get merged
 
-join the #krebs channel on freenode.
-If you don't know anything about irc,  you can just use this webirc: https://webchat.freenode.net/#krebs
-If you are not known, introduce yourself, explain what you are doing and what you want to contribute. upload your patch to some pastebin (preferably https://paste.krebsco.de) or send it to spam@krebsco.de
+Join the #krebs channel on freenode.
+If you don't know anything about IRC, you can just use this webirc: https://webchat.freenode.net/#krebs
+If you are not known, introduce yourself, explain what you are doing and what you want to contribute.
+Then send the formatted patch either sent via email to spam@krebsco.de, or
+upload it to some pastebin (preferably https://paste.krebsco.de) and share the
+link to the pasted patch in #krebs.
+Now some patience will be required, but you will soon receive confirmation or
+change requests for your patch, and it will get deployed to the super nodes.
+At this point your host will be able to join the network.
```

Comment:
typically, you will get more/faster attention when you ping the right people. I think it would make sense to tell users whom to @-message in the channel because those persons have commit-rights